### PR TITLE
Add Members of Cabinet section

### PR DESCRIPTION
**Pull request by Wisawing**
For #70 
Add cabinet member list categorized by ministry.
- Add CabinetMember component for displaying each member in the cabinet.
- Add CabinetMemberList for displaying list of members in one ministry.
- Add CabinetMemberSection for displaying all members in all ministry.

- Need access to invision inspect mode for a more detailed styling.
- Not sure if the structure of the file is correct. Please review.

### CODE REVIEW

File: src/components/cabinetMemberSection/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "กระทรวงวัฒนธรรม",
+    "กระทรวงอุตสาหกรรม",
+    "กระทรวงแรงงาน",
+  ]
+
+  // president need special filter
+  const presidentMinistry = "นายกรัฐมนตรี"
+  const presidentList = data.allPeopleYaml.edges.filter(people =>
+    people.node.cabinet_position.some(
+      peoplePostion => peoplePostion === presidentMinistry
```
Comment (rapee) [inline src/components/cabinetMemberSection/index.js]: Should be `peoplePosition`

**Comment by rapee** [comment]
Thanks for this PR @Wisawing 👍 

1. Make names on cards a link to their own page e.g. `/people/ประยุทธ์-จันทร์โอชา`

2. Besides above comments, I have one more feature request.

On each of member cards, can you insert a second line saying "รัฐมนตรีว่าการกระทรวงการคลัง" or "รัฐมนตรีช่วยว่าการกระทรวงการคลัง". I totally forgot about this. Please make sure that "รัฐมนตรี..." always comes before "รัฐมนตรีช่วย...".

**Comment by rapee** [comment]
@Wisawing can you help with above comments?

**Comment by Wisawing** [comment]
@rapee I can, just that I haven't get to it yet. Should be able to do this on Friday. Do you need it before that?

**Comment by Wisawing** [comment]
@rapee Fixed according to comments.
May I also suggest making the whole card as a link instead of the name?

**Comment by rapee** [comment]
Yes. I'll create a new issue for that
> May I also suggest making the whole card as a link instead of the name?

**Comment by rapee** [comment]
Thanks for your contribution! 👍  @Wisawing

**Review by ThanaphonKonjanakraisit — COMMENTED** [review]
Best@GitHub.co.BTC.ECH.th``

**Pull request merged**